src/app.py

```diff
@@ -63,12 +63,19 @@ def populate_mock_dynamo():
     with open('mock_experiment.json') as json_file:
         experiment_data = json.load(json_file, use_decimal=True)
 
+    with open('multisample_mock.json') as json_file:
```

Comment:
we also need to modify the file upload part to get the multiple sample experiment count matrix too